src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -2370,6 +2372,32 @@ public void testBuild() {
         assertEquals(BuildStatus.BUILT_INVALID, networkModificationTreeService.getBuildStatus(modificationNode5.getId()));
     }
 
+    @Test
+    public void deleteModificationRequest() {
+        createStudy("userId", CASE_UUID, false);
+        UUID studyNameUserIdUuid = studyRepository.findAll().get(0).getId();
+        UUID rootNodeUuid = getRootNodeUuid(studyNameUserIdUuid);
+        NetworkModificationNode modificationNode = createNetworkModificationNode(rootNodeUuid);
+        UUID modificationUuid = UUID.randomUUID();
+        webTestClient.delete()
+            .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/{modifcationUuid}", studyNameUserIdUuid, modificationNode.getId(), modificationUuid)
+            .exchange()
+            .expectStatus().isOk();
+
+        assertTrue(getRequestsDone(1).stream().anyMatch(
+            r -> r.matches("/v1/groups/" + modificationNode.getNetworkModification() + "/modifications/" + modificationUuid))
+        );
+
+        var res = output.receive(1000);
+        assertEquals(studyNameUserIdUuid, res.getHeaders().get("studyUuid"));
+        var nodesList = res.getHeaders().get("nodes", List.class);
+        assertNotNull(nodesList);
+        assertEquals(1, nodesList.size());
+        assertEquals(modificationNode.getId(), nodesList.get(0));
+        assertEquals("nodeUpdated", res.getHeaders().get("updateType"));
```

Comment:
maybe add tests for others `updateType` values ?

Reply:
? The only value updateType can be is nodeUpdated, if it is equals then it can't be another value.